docs/howto/how-to-build-package.md

```diff
@@ -1 +1,170 @@
 # How to Build Package
+
+## Overview
+
+This document describes how to build a Package to run the model in our runtime
+_onert_ that consists of model and additional file(s). Users can build a
+package through command line tools.
+
+Steps of building a Package:
+1. Import model and convert to circle
+1. Optionally, optimize and quantize circle
+1. Create package from circle
+
+NOTE: Examples and options of each command shown below are from the version of
+writing this document. They may differ from latest version of commands.
+Please fire an issue or post a PR to correct them if anything needs update.
+
+## Import model
+
+Currently TensorFlow and TensorFlow lite models are supported as of writing
+this document.
+
+To import a model, use `one-import` with a model framework key and arguments.
+```
+$ one-import FRAMEWORK [arguments]
+```
+
+Execute `one-import` without any key will show the list of supported frameworks.
+
+Example of `one-import` command:
+```
+$ one-import
+Usage: one-import [FRAMEWORK] ...
+Available FRAMEWORK drivers:
```

Comment:
@seanshpark , It's not something that needs to be corrected this time, but wouldn't it be better to be FORMAT instead of FRAMEWORK? It doesn't matter whether 'tf' and 'tflite' are FRAMEWORK or FORMAT, but 'bcq' seems to be more suitable for FORMAT, and in the case of 'tf', a 'saved model' or 'keras model', etc. can be added later.

Reply:
Yes, when I started with using `FRAMEWORK`, it was something like framework and then later `bcq` was added.
And this seems to be more like a format. I'll revise this after this lands. :)